packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.styles.ts

```diff
@@ -0,0 +1,64 @@
+import { createStyles, keyframes } from '$lib/styles';
+import type { SvelteUINumberSize, SvelteUIColor, ColorShades } from '$lib/styles';
+
+export interface BreadcrumbItemStylesParams {
+	color: SvelteUIColor;
+	size: SvelteUINumberSize;
+}
+
+
+export const sizes = {
+	xs: 3,
+	sm: 5,
+	md: 8,
+	lg: 12,
+	xl: 16
+};
+
+export const fontSizes = {
+	xs: 12,
+	sm: 14,
+	md: 16,
+	lg: 18,
+	xl: 20
+};
+
+export default createStyles(
+	(theme, { color, size }: BreadcrumbItemStylesParams) => ({
+		root: {
+			[`${theme.dark} &`]: {
+				backgroundColor: theme.fn.themeColor('dark', 4)
```

Comment:
Hum, I don't think it should contain the change in background color. The only thing that should change with the change in theme is the color of the separator